swap_meet/item.py

```diff
@@ -1,2 +1,19 @@
 class Item:
-    pass
+    def __init__(self, category='', condition = 0):
+        self.category=category
+        self.condition = condition
+    def __str__(self):
+        return f"Hello World!"
+    def condition_description(self):
+        if self.condition<=0:
+            return f"bad condition"
+        elif self.condition<=1:
+            return f"Far below Average Condition."
+        elif self.condition<=2:
+            return f"Below Average Condition."
+        elif self.condition<=3:
+            return f"average."
+        elif self.condition<=4:
+            return f"Above average."
+        elif self.condition<=5:
```

Comment:
Because you are using an if statement this last line can just use `else`
```suggestion
        else:
```